### PR TITLE
[IMP] account_edi: put a search filter for what is being processed + …

### PR DESCRIPTION
**Pull request by jco-odoo**
…show error msg in list view

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jco-odoo** [comment]
@smetl An old branch, but it might still be interesting to see if we can add something like this in master (e.g. in case of manual processing)

File: addons/account_edi/models/account_move.py [hunk 1]
```diff
@@ -14,17 +14,17 @@ class AccountMove(models.Model):
...
-        string="Electronic invoicing",
+        string="E-invoice state",
         store=True,
         compute='_compute_edi_state',
         help='The aggregated state of all the EDIs with web-service of this move')
     edi_error_count = fields.Integer(
         compute='_compute_edi_error_count',
         help='How many EDIs are in error for this move ?')
-    edi_blocking_level = fields.Selection(
+    edi_blocking_level = fields.Selection(string="E-invoice Blocking Level",
```
Comment (smetl) [inline addons/account_edi/models/account_move.py:24]: align this with others attributes

File: addons/account_edi/models/account_move.py [hunk 2]
```diff
@@ -14,17 +14,17 @@ class AccountMove(models.Model):
...
         selection=[('info', 'Info'), ('warning', 'Warning'), ('error', 'Error')],
         compute='_compute_edi_error_message')
-    edi_error_message = fields.Html(
+    edi_error_message = fields.Html(string="E-invoice Error Message",
         compute='_compute_edi_error_message')
```
Comment (smetl) [inline addons/account_edi/models/account_move.py:28]:
```suggestion
    edi_error_message = fields.Html(
        string="E-invoice Error Message",
        compute='_compute_edi_error_message',
    )
```

**Comment by jco-odoo** [comment]
@lordkrandel Maybe some inspiration

**Comment by smetl** [comment]
@lordkrandel Don't be inspired too much by this PR since @jco-odoo is not doing anything on it since few months even if the review is already done. If you are interested by this PR, feel free to end it yourself ;)

**Comment by jco-odoo** [comment]
@smetl https://github.com/odoo/odoo/pull/113577

**Pull request closed without merging**